### PR TITLE
Arodriguez/feature 184 update report error

### PR DESCRIPTION
**Pull request by ALFAROSO**
Hi guys this is the issue I had on trello

Issue:
issue

Fix:
done

### CODE REVIEW

File: pages/report-hazard/script.js [hunk 1]
```diff
@@ -1001,3 +1002,12 @@ reportHazardForm.addEventListener('scroll', function (event) {
     continueBtnContainer.style.boxShadow = 'none';
   }
 });
+
+//imprimr currentReport siempre
```
Comment (ReinhardtBotha) [inline pages/report-hazard/script.js:1006]: Please stick to only English

File: pages/report-hazard/script.js [hunk 2]
```diff
@@ -1001,3 +1002,12 @@ reportHazardForm.addEventListener('scroll', function (event) {
...
+function handleHashChange() {
+  var newUrl = window.location.hash;
+  console.log('Nueva URL:', newUrl);
+  console.log(currentReport);
```
Comment (ReinhardtBotha) [inline pages/report-hazard/script.js:1010]: Remove the console logs

**Review by ReinhardtBotha — CHANGES_REQUESTED** [review]
Looks good Alberth, just have a look at my 2 comments.

**Review by whamester — CHANGES_REQUESTED** [review]
https://github.com/whamester/WMDD-4885-Project/assets/57273227/87186966-d667-4456-9f5b-a664a9711b85

There is something weird when adding new images.

File: pages/report-hazard/script.js (hunk 1 above)
Comment (ALFAROSO) [reply pages/report-hazard/script.js:1006]: Sorry that is just for testing purposes I should have deleted everything after line 1005

File: pages/report-hazard/script.js (hunk 2 above)
Comment (ALFAROSO) [reply pages/report-hazard/script.js:1010]: Sorry that is just for testing purposes I should have deleted everything after line 1005

**Pull request closed without merging**